vuestorefront/magento2
# fix(theme): fix adding product to a wishlist from a listed category page

## Description
Fix adding product to a wishlist from a listed category page

## Related Issue
#444 

## Motivation and Context
bugfix

## How Has This Been Tested?
---
1. Visit category page
2. Change view from a grid to list
3. Use add to wishlist action to add or remove product from wishlist
4. Observe if product is added/removed and if action name changed accordingly
---
1. Go to category page
2. Add product to the wishlist from a grid view
3. Switch to list view
4. Observe if action is properly displayed

## Screenshots (if appropriate):

## Types of changes


## Checklist:
